test/object_storage/test_object_storage.py

```diff
@@ -72,15 +74,15 @@ def test_copy(transfer_type: str, request: Any) -> None:
 def test_copy_local_files_from(tmp_path: Path) -> None:
     source = LocalTransfer(tmp_path / "source", prefix="s-prefix")
     destination = LocalTransfer(tmp_path / "destination", prefix="d-prefix")
+    mock_progress_fn = MagicMock(return_value=None)
 
     source.store_file_from_memory("some/a/key.ext", b"content_a", metadata={"info": "aaa"})
     source.store_file_from_memory("some/b/key.ext", b"content_b", metadata={"info": "bbb"})
-    destination.copy_files_from(
-        source=source,
-        keys=["some/a/key.ext", "some/b/key.ext"],
-    )
+    destination.copy_files_from(source=source, keys=["some/a/key.ext", "some/b/key.ext"], progress_fn=mock_progress_fn)
     assert destination.get_contents_to_string("some/a/key.ext") == (b"content_a", {"info": "aaa", "Content-Length": "9"})
     assert destination.get_contents_to_string("some/b/key.ext") == (b"content_b", {"info": "bbb", "Content-Length": "9"})
+    assert mock_progress_fn.call_count == 2
```

Comment:
Why not test the actual keys were called here?

Reply:
Replaced with simply
```python
assert mock_progress_fn.call_args_list == [mock.call("some/a/key.ext"), mock.call("some/b/key.ext")]
```

Previous assertion  `mock_progress_fn.assert_has_calls` did not raise when the call arg list included additional calls.